prov/efa/src/rxr/rxr_pkt_entry.c

```diff
@@ -182,6 +182,12 @@ void rxr_pkt_entry_release_rx(struct rxr_ep *ep,
 		else
 			ep->rx_bufs_efa_to_post++;
 	}
+
+	if (pkt_entry->type == RXR_PKT_ENTRY_READ_COPY) {
```

Comment:
there are a lot of if statements in here; is this really a good path to continue for something that is in the critical path?

Reply:
Unfortunately, the caller of this function does not know the ```pkt_entry->type``` either, so there has to be a branch. This branch is just for monitoring ```rx_readcopy_pkt_pool``` usage, maybe I can put it in debug?